src/pages/docs/guides/chat/build-task-oriented.mdx

```diff
@@ -0,0 +1,419 @@
+Features such as efficient connection management are available to help you reduce costs.
+
+## Architecting your task-oriented chat: UX first, scale always
+
+The most important decision when developing task-oriented chat is the experience you want users to have within their specific context. This will determine how chat integrates with your task flow and ultimately how effectively it enhances the user experience.
+
+**What are task-oriented chats?** Any chat that serves as a companion to a specific activity or workflow: resolving a support ticket, coordinating a delivery, discussing a chess game, collaborating on a project, or facilitating a marketplace transaction. These chats are typically short-lived (minutes to days) and highly contextual. They can be 1:1 or small group conversations.
+
+For large-scale chat scenarios with many users per room, see our [building livestream chat guide](/docs/guides/chat/build-livestream).
+
+**Task-oriented room architecture:** The optimal approach is one chat room per task, where all participants join to communicate within that specific context. Rooms are:
+
+* **Automatically scaled** - Ably handles any number of concurrent rooms without provisioning.
+* **History-enabled** - messages stored for 30 days by default (configurable up to a year on request).
+* **Kept under control** - [capabilities](/docs/chat/setup#authentication) let you control who can join and the level of access to features and actions.
+
+Each room includes all task participants: customers and agents in support, buyers and sellers in marketplaces, players in games, team members in projects. Participants can join and leave as the task evolves. Access to message history and other features can be granted to all or select participants via capabilities.
+
+## Message throughput and rate management
+
+In task-oriented chat, each room is typically low throughput. Scaling comes by running as many rooms as you need at the same time. Ably handles everything from 1:1 conversations and small group discussions to [livestream chats](/docs/guides/chat/build-livestream) with the same reliability and scalability guarantees.
+
+Chat rooms scale horizontally. Ably allows you to have as many rooms as you need running in parallel.
+
+* **Proven at scale:** Ably delivers over 500 million messages per day for customers.
+* **Rate limiting:** Prevent spam and maintain conversation quality with global or per-user throttling appropriate for your use case.
+
+## Authentication
+
+Authentication is a design decision. 
+
+In some cases, you'll want to tie participants to their identity in your application.
+In others, anyone can join and participate.
+Ably Chat supports both of these scenarios - you don't need to go through the hassle of setting up
+users in Ably and trying to keep them in sync with your application. Every client has a `clientId`.
+If you want to allow random users, just use a random string. If you want to tie it to your application,
```

Comment:
Nit: a few places here we've done heavily wrapped lines, in others we keep paragraphs to one line (which I think is the preferred), can we be consistent?